serverless/networking/serverless-ossm.adoc

```diff
@@ -10,80 +10,13 @@ toc::[]
 Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options that are not supported when using {ServerlessProductName} with the default Kourier implementation.
 These options include setting custom domains, using TLS certificates, and using JSON Web Token authentication.
 
-.Prerequisites
-. Install the xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessOperatorName}] and xref:../../serverless/installing_serverless/installing-knative-serving.adoc#installing-knative-serving[Knative Serving].
-. Install xref:../../service_mesh/v1x/preparing-ossm-installation.adoc#preparing-ossm-installation-v1x[{ProductName}].
-
-.Procedure
-. Add the `default` namespace to the xref:../../service_mesh/v1x/installing-ossm.adoc#ossm-member-roll-create_installing-ossm[ServiceMeshMemberRoll] as a member:
-+
-
-[source,yaml]
-----
-apiVersion: maistra.io/v1
-kind: ServiceMeshMemberRoll
-metadata:
-  name: default
-  namespace: istio-system
-spec:
-  members:
-    - default
-----
-
-+
-[IMPORTANT]
-====
-Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
-====
-
-. Create a network policy that permits traffic flow from Knative system Pods to Knative services:
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving` namespace:
-+
-
-[source,terminal]
-----
-$ oc label namespace knative-serving serving.knative.openshift.io/system-namespace=true
-----
+[id="serverless-ossm-prerequisites"]
+== Prerequisites
 
-.. Add the `serving.knative.openshift.io/system-namespace=true` label to the `knative-serving-ingress` namespace:
-+
-
-[source,terminal]
-----
-$ oc label namespace knative-serving-ingress serving.knative.openshift.io/system-namespace=true
-----
-
-.. Copy the following NetworkPolicy resource into a YAML file:
-+
-
-[source,yaml]
-----
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: allow-from-serving-system-namespace
-  namespace: default
-spec:
-  ingress:
-  - from:
-    - namespaceSelector:
-        matchLabels:
-          serving.knative.openshift.io/system-namespace: "true"
-  podSelector: {}
-  policyTypes:
-  - Ingress
-----
-
-.. Apply the NetworkPolicy resource:
-+
+. Install the xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessOperatorName}] and xref:../../serverless/installing_serverless/installing-knative-serving.adoc#installing-knative-serving[Knative Serving].
+. Install {ProductName}.
```

Comment:
There are two versions of Service Mesh docs now; there's no way to link to a main install section, just 1x or 2x, so that's why I removed this. If we include links to both that opens a can of worms about which we recommend for which Serverless versions etc and I don't think that's something we should be on the hook to discuss / maintain here in the docs. They should be able to find these easily enough and we can discuss it again if it causes any problems for users.